### ISSUE TITLE
Improve meta-property documentation

### ISSUE THREAD

**Issue by twilmes** [JanusGraph/janusgraph#225]
Section 26.3 of the docs discusses meta-properties but not by name. I think this may be confusing for folks jumping between this and the TinkerPop docs. Perhaps we could update the docs to specifically reference meta-properties and extend the example.

**Comment by pluradj** [comment]
Agreed, as easy as it may be to refer back to the TinkerPop docs, I think the JanusGraph docs need to be able to stand alone in as many places as possible. This is especially important as the most recent TinkerPop version could potentially get way ahead of what JanusGraph bundles.
